content/20.treatments.md

```diff
@@ -540,19 +540,59 @@ Although exciting, the cost of vaccine manufacturing and electroporation may mak
 
 #### RNA Vaccines
 
-Brief background on the therapeutic.
+RNA vaccines are nucleic-acid based modalities that code for viral antigens against which the human body elicits a humoral and cellular immune response. 
+The mRNA technology is transcribed _in vitro_ and delivered to cells via lipid nanoparticles (LNP). 
+They are recognized by ribosomes in vivo and then translated and modified into functional proteins [@doi:10.3389/fimmu.2019.00594]. 
+The resulting intracellular viral proteins are displayed on surface MHC I proteins, provoking a strong CD8+ T cell response as well as a CD4+ T cell and B cell-associated antibody responses [@doi:10.3389/fimmu.2019.00594]. 
+
+Naturally, mRNA is not very stable and can degrade quickly in the extracellular environment or the cytoplasm. 
+The LNP covering protects the mRNA from enzymatic degradation outside of the cell [@doi:10.4155/tde-2016-0006]. 
+Codon optimization to prevent secondary structure formation and modifications of the poly-A tail as well as the 5’ untranslated region to promote ribosomal complex binding can increase mRNA expression in cells. 
+Furthermore, purifying out dsRNA and immature RNA with FPLC (fast performance liquid chromatography) and HPLC (high performance liquid chromatography) technology will improve translation of the mRNA in the cell [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245].
+
+mRNA vaccines confer many advantages over traditional viral vectored vaccines and DNA vaccines. 
+In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, and thereby reducing the risk of a detrimental immune response in the host [@doi:10.1016/j.immuni.2020.03.007]. 
+Unlike DNA vaccines, mRNA technologies are naturally degradable and non-integrating, and they do not need to cross the nuclear membrane in addition to the plasma membrane for their effects to be seen [@doi:10.3389/fimmu.2019.00594].  
+Furthermore, mRNA viruses are easily, affordably, and rapidly scalable.
+
+mRNA vaccines have been developed for therapeutic and prophylactic purposes. 
+mRNA vaccines have not been licensed or commercialized; however, several agents for cancer and infectious diseases, including rabies, coronavirus, influenza, rabies, and cytomegalovirus, have shown promising immunization in animal models and are currently in clinical trials [@url:https://clinicaltrials.gov/ct2/show/NCT01669096?term=mRNA+vaccine&draw=7]. 
```

Comment:
This might flow a bit better if these two sentences were reworded a bit (just a suggestion, feel free to ignore if you disagree). Perhaps something like:

Although mRNA vaccines have been developed for therapeutic and prophylactic purposes, none have been licensed or commercialized thus far. Nevertheless, they have show promise in animal models and preliminary clinical trials for several indications, including rabies, coronavirus, influenza, rabies, and cytomegalovirus.

Reply:
Thank you this reads much nicer!